components/form-toggle/index.js

```diff
@@ -21,27 +22,35 @@ class FormToggle extends Component {
 	}
 
 	render() {
-		const { className, checked, children, onChange = noop } = this.props;
-		const id = 'toggle-' + this.id;
+		const {
+			className,
+			checked,
+			onChange = noop,
+			showHint = true,
+			id = 'toggle-' + this.id,
```

Comment:
I think we should treat it as unique from the parent to allow its usage for the label `htmlFor` without the need to "know" the prefix.